src/main/java/picard/util/IntervalListTools.java

```diff
@@ -495,6 +514,22 @@ private IntervalList openIntervalLists(final List<File> files, BinaryOperator<In
                 .orElse(null);
     }
 
+    private static IntervalList uniqued(IntervalList non_unique, boolean merge_abutting) {
```

Comment:
I'd add a "//TODO:'' item to move this to htsjdk and use that once we can. (possibly adding issues that can be referenced)